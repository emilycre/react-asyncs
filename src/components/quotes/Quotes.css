.Quotes {
  box-sizing: border-box;
  display: flex;
  justify-content: space-around;
  height: 100%;
  flex-wrap: wrap;
  align-items: center;
  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;

  li {
    width: 18vw;
    margin: 2em;
    padding: 2rem;
    background-color: rgb(255, 185, 35);
    border-radius: 5px;
    list-style-type: none;
  }

  section {
    margin: auto;
    width: 100%;
    animation: insert 0.5s linear;

  img {
    object-fit: scale-down;
    width: 15rem;
    height: 15rem;
    margin: auto;
  }
  
  @keyframes insert {
    from {
      width: 0;
    }

    to {
      width: 100%;
    }
  }
}
}
