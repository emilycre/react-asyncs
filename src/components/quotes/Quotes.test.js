import React from 'react';
import { shallow } from 'enzyme';
import Quotes from './Quotes';

describe('Quotes Component', () => {
  it('Renders a Quotes', () => {
    const quotes = [
      {
        character: 'Fry',
        quote: '...',
        image: '/image'
      },
      {
        character: 'Fry',
        quote: '!',
        image: '/image'
      }
    ];
    const wrapper = shallow(<Quotes quotes={quotes} />);
    expect(wrapper).toMatchSnapshot();
  });
});
