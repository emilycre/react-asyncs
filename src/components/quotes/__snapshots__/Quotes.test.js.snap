// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes Component Renders a Quotes 1`] = `
<ul
  className="Quotes"
>
  <li
    key="0"
  >
    <Quote
      quote={
        Object {
          "character": "Fry",
          "image": "/image",
          "quote": "...",
        }
      }
    />
  </li>
  <li
    key="1"
  >
    <Quote
      quote={
        Object {
          "character": "Fry",
          "image": "/image",
          "quote": "!",
        }
      }
    />
  </li>
</ul>
`;
